packages/strapi-helper-plugin/lib/src/utils/auth.js

```diff
@@ -0,0 +1,104 @@
+import { isEmpty } from 'lodash';
+
+const TOKEN_KEY = 'jwtToken';
+const USER_INFO = 'userInfo';
+
+const parse = JSON.parse;
+const stringify = JSON.stringify;
+
+const auth = {
+  clearAppStorage() {
+    if (localStorage) {
+      return localStorage.clear();
+    }
+
+    if (sessionStorage) {
+      return sessionStorage.clear();
+    }
+
+    return null;
+  },
+
+  clearToken(tokenKey = TOKEN_KEY) {
```

Comment:
A lot of duplicated code in this file already using dynamic parameters. Pretty sure it could be merged...